internal/app/out.go

```diff
@@ -0,0 +1,76 @@
+package app
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io"
+	"os"
+
+	"github.com/sapcc/concourse-netbox-resource/internal/concourse"
+	"github.com/sapcc/concourse-netbox-resource/internal/helper"
+)
+
+var (
+	UsageOut string = `This command implements the Concourse out interface as a noop. It reads the input, validates it, and outputs the version.
+
+	Example: out /tmp/build/put < source.json
+	`
+)
+
+func Out() {
+	var (
+		input  concourse.Input
+		output concourse.Output
+		err    error
+	)
+
+	input, _, err = validateOutInput(os.Stdin)
+	if err != nil {
+		fmt.Fprintln(os.Stderr, fmt.Errorf("input validation failed: %w", err))
+		os.Exit(1)
+	}
+
+	output.Version = input.Version
+	if err := json.NewEncoder(os.Stdout).Encode(output); err != nil {
+		fmt.Fprintln(os.Stderr, fmt.Errorf("failed to write JSON to stdout: %w", err))
+	}
+}
+
+func validateOutInput(stdin *os.File) (concourse.Input, concourse.Input, error) {
```

Comment:
Same story about `io.Reader` and `stdin`.

Reply:
done